e2e/actors/base.js

```diff
@@ -22,7 +22,7 @@ let currentUser = {};
 
 module.exports = {
   async signIn(user) {
-    if (currentUser !== user) {
+    if (this.isPuppeteer() && currentUser !== user) {
```

Comment:
when run in SauceLab via WebDriver, each test is started in new browser, so this optimalisation cause test failures